### ISSUE TITLE
Library Request: cde

### ISSUE THREAD

**Issue by AlexCatarino** [QuantConnect/Lean#7902]
#### Expected Behavior
[cde](https://pypi.org/project/cde/) implements various methods for conditional density estimation

- Parametric neural network based methods
  - Mixture Density Network (MDN)
  - Kernel Mixture Network (KMN)
  - Normalizing Flows (NF)
- Nonparametric methods
  - Conditional Kernel Density Estimation (CKDE)
  - Neighborhood Kernel Density Estimation (NKDE)
- Semiparametric methods
  - Least Squares Conditional Density Estimation (LSKDE)

#### Checklist
- [x] I have completely filled out this template
- [x] I have confirmed that this issue exists on the current `master` branch
- [x] I have confirmed that this is not a duplicate issue by searching [issues](https://github.com/QuantConnect/Lean/issues)

**Comment by Martin-Molinero** [comment]
`cde` currently supports `tensorflow<=1.7.0 and >=1.4` which is really old (2018) was last updated in 2019 & does not seem to be public, closing for now
